db.yaml

```diff
@@ -0,0 +1,4 @@
+mysql_host: 'localhost'
+mysql_user: 'root'
+mysql_password: 'Marija01101984'
+mysql_db: 'flaskapp'
```

Comment:
Important note: this file contains secrets (passwords, database credentials, etc). It should NOT be on Github. You can add a note in the README about how to create such a file (or put an example file on Github without the actual values) but remove this file from version control (git).